src/main/java/com/mycompany/myapp/service/CSV/ParsingContainer.java

```diff
@@ -0,0 +1,93 @@
+package com.mycompany.myapp.service.CSV;
+
+import com.mycompany.myapp.domain.Pool;
+
+import java.util.LinkedList;
+
+public class ParsingContainer {
+    private String poolId;
+    private String displayName;
+    private Integer maximumCount;
+    private Boolean enabled;
+    private String description;
+
+    public ParsingContainer() {
```

Comment:
Do we follow any convention with fields and constructors? We could use the one from DTO

Reply:
Yup, this constructor is excessive.